sdk/include/aditof/camera_definitions.h

```diff
@@ -30,6 +31,33 @@ enum class ConnectionType {
     LOCAL     //!< The device is embedded in the system
 };
 
+/**
+ * @struct IntrinsicParameters
+ * @brief Provides the intrinsic parameters of a camera.
+ */
+struct IntrinsicParameters {
+	/**
+	 * @brief The intrinsic 3x3 parameter matrix (K) with values specified in pixel units.
```

Comment:
What is 'K'?

Reply:
K is the letter used by Hartley and Zisserman to denote the matrix containing the intrinsic parameters.
This matrix is known by many different names (intrinsic matrix, K matrix, internal camera matrix etc), so I wanted to give more details in order to make sure it is clear what it contains. 
Should I replace the K in the description with something like "K = [fx, 0, cx, 0, fy, cy, 0, 0, 1]" or just delete it altogether?

Reply:
Thanks for explaining. The 'K' alone can be confusing. Maybe say "K matrix" or "a.k.a. K Matrix")
You can add  "K = [fx, 0, cx, 0, fy, cy, 0, 0, 1]" if you think it helps, but then you need to explain what fx, cx, etc. are.

Reply:
solved in commit https://github.com/analogdevicesinc/aditof_sdk/pull/203/commits/adba4901642e22b8626a1910e3c23271c2af6dc3